Fluid/Values/ArrayValue.cs

```diff
@@ -145,15 +145,24 @@ public override bool Equals(object obj)
             // The is operator will return false if null
             if (obj is ArrayValue otherValue)
             {
-                return Values.Equals(otherValue.Values);
+                return Equals(otherValue);
             }
 
             return false;
         }
 
         public override int GetHashCode()
         {
-            return Values.GetHashCode();
+            var hc = new HashCode();
```

Comment:
I think it's ok to add a reference to Microsoft.Bcl.HashCode instead for netstandard2.0. There is already a section in Directory.Packages.props for this TFM